src/types/institution.types.ts

```diff
@@ -1,46 +1,39 @@
-export interface InstitutionName {
+export interface RecursiveUnit {
   name: string;
-  language: string;
+  id: string;
+  subunits?: RecursiveUnit[];
 }
 
-export const emptyInstitutionNames: InstitutionName[] = [
-  {
-    name: '',
-    language: '',
-  },
-];
+export interface Subunit {
+  name: string;
+  id: string;
+}
 
-export interface Institution {
-  title?: string;
-  institutionNames: InstitutionName[];
-  acronym: string;
-  cristinInstitutionId: string;
-  country: string;
-  cristinUnitId: string;
+export interface FormikUnit {
+  name: string;
+  id: string;
+  subunits: Subunit[];
+  unit: RecursiveUnit;
 }
 
-export const emptyInstitution: Institution = {
-  title: '',
-  institutionNames: emptyInstitutionNames,
-  acronym: '',
-  cristinInstitutionId: '',
-  country: '',
-  cristinUnitId: '',
-};
+export type Unit = Pick<FormikUnit, 'name' | 'id' | 'subunits'>;
```

Comment:
trenger vi to ulike typer for `Unit` og `RecursiveUnit`? de er veldig like

Reply:
subunits har forskjellig type